templates/reactor-config-secret.yml.j2

```diff
@@ -14,6 +14,11 @@ clusters:
 
 {% if osbs_odcs_enabled %}
 odcs:
+    {% if osbs_odcs_api_url %}
```

Comment:
I think it might be safe to remove this conditional. If user doesn't define a value for `osbs_odcs_api_url`, it'll just output an empty `api_url` which is harmless.